pkg/applyinator/applyinator.go

```diff
@@ -146,7 +146,8 @@ func (a *Applyinator) Apply(ctx context.Context, cp CalculatedPlan) ([]byte, err
 	for index, instruction := range cp.Plan.Instructions {
 		logrus.Debugf("Executing instruction %d for plan %s", index, cp.Checksum)
 		executionInstructionDir := filepath.Join(executionDir, cp.Checksum+"_"+strconv.Itoa(index))
-		output, err := a.execute(ctx, executionInstructionDir, instruction)
+		prefix := cp.Checksum + "_" + strconv.Itoa(index)
```

Comment:
Do we know for certain `index` will always have an integer value in it?

Reply:
I believe so, `cp.Plan.Instructions` is a slice